src/main/java/uk/gov/hmcts/reform/blobrouter/services/storage/BulkScanSasTokenCache.java

```diff
@@ -45,6 +45,10 @@ public String getSasToken(String containerName) {
         return tokenCache.get(containerName, c -> this.createSasToken(c));
     }
 
+    public void removeFromCache(String containerName) {
+        tokenCache.invalidate(containerName);
+    }
```

Comment:
What will call this method and when?

Reply:
later we will remove sas token from cache if we get any 400 errors.

Reply:
I'm asking because I wonder if it would be better to use the existing method that gets the SAS token and add a parameter that decides if the cache item should be recreated.

Reply:
we are not going to recreate if we got error.